File: FR_08_02_korekta/korekta.r
f <- file("stdin")
open(f)

t <- as.integer(readLines(f, n=1))

while (t>0) {
    tab <- as.numeric(strsplit(readLines(f, n=1)," ")[[1]])
    wj <- 2*(tab[1]+tab[2])-tab[1]-tab[3]-tab[5]
    pj <- tab[1]+tab[2]-wj
    write(paste(as.character(wj)," ",as.character(pj), sep=""), stdout())

}
close(f)

File: FR_08_02_korekta/korekta.f90
program korekta
implicit none;

integer :: t 
integer :: wj, ws, ww, wg , pj, ps, pw, pg
integer :: i

read(*,*) t
do i = 1,t 
    read(*,*) ws, ps, ww, pw, wg, pg 
    wj = (2*(ws+ps) - ws-ww-wg)
    pj = (ws+ps) - wj
    write(*,'(I0, a1, I0)') wj, ' ', pj
end do 

end program